test/WebJobs.Script.Tests.Integration/NodeEndToEndTests.cs

```diff
@@ -1229,6 +1229,12 @@ public void ExcludedFunction_NotAddedToHost()
             Assert.Equal(1, array[1]);
         }
 
+        public void Dispose()
+        {
+            // Make sure all logs are flushed before next test.
```

Comment:
If this works, I'll move it to the DeleteLogs() call, which is where I'm having the problem. I delete, expecting a clean slate -- but then the flush writes out logs from the previous test just after the file is deleted. This really only applies to the "Scenarios" test where we re-use the same function for multiple scenarios (and thus the same file). But it's probably generally useful when cleaning up to make sure that you're truly cleaning up everything.